preview-env/create/action.yml

```diff
@@ -105,11 +105,11 @@ runs:
       ARGOCD_AUTH_TOKEN: ${{ inputs.argocd_token }}
       ARGOCD_OPTS: "--grpc-web"
       ARGOCD_SERVER: ${{ inputs.argocd_server }}
-  - name: Wait for sync (up to 20 minutes)
+  - name: Wait for sync (up to 2 minutes)
```

Comment:
I assume this is a leftover, used for testing, which must now must be removed ... ?